### PR TITLE
dev, v3.0: add maintain kubernetes node file

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? 

1. add maintain kubernetes node file
2. update TOC

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1417/
https://github.com/pingcap/docs-cn/pull/1538

### Which version does your change affect? 

dev, v3.0

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/how-to/maintain/tidb-in-kubernetes/tidb-node.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+    ```
+
+    When you confirm that all pods are running normally, then you have successfully finished the maintenance task.
+
+## Maintain nodes that hold TiKV instances
+
+Migrating TiKV instances is relatively slow and might lead to unwanted data migration load on the cluster. For this reason, you need to choose different strategies as needed prior to maintaining the nodes that hold TiKV instances:
+
+- If you want to maintain a node that is recoverable in a short term, you can recover the TiKV node from where it is after the maintenance without migrating it elsewhere.
+-  If you want to maintain a node that is not recoverable in a short term, you need to make a plan for the TiKV migration task.
```
Comment (yikeke) [inline dev/how-to/maintain/tidb-in-kubernetes/tidb-node.md]:
```suggestion
- If you want to maintain a node that is not recoverable in a short term, you need to make a plan for the TiKV migration task.
```

File: dev/how-to/maintain/tidb-in-kubernetes/tidb-node.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+{{< copyable "shell-regular" >}}
+
+```shell
+pd-ctl -d config set max-store-down-time 10m
+```
+
+After configuring `max-store-down-time` to an appropriate value, the follow-up operations are the same as those in [Maintain nodes that hold PD and TiDB instances](#maintain-nodes-that-hold-pd-and-tidb-instances).
+
+### Maintain an irrecoverable node in a short terminform
```
Comment (yikeke) [inline dev/how-to/maintain/tidb-in-kubernetes/tidb-node.md]:
```suggestion
### Maintain an irrecoverable node in a short term
```

File: dev/how-to/maintain/tidb-in-kubernetes/tidb-node.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+# Maintain Kubernetes Nodes that Hold the TiDB Cluster
+
+TiDB is a highly available database that can run smoothly when some of the database nodes go offline. For this reason, you can safely shut down and maintain the Kubernetes nodes at the bottom layer without influencing TiDB's service. Specifically, you need to adopt various maintenance strategies when handling nodes that hold PD, TiKV, and TiDB instances because of their different features.
+
+This document introduces how to perform a temporary or long-term maintenance task for the Kubernetes nodes.
+
+## Prerequisites
+
+- [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [`tkctl`](/tidb-in-kubernetes/tkctl.md)
```
Comment (yikeke) [inline dev/how-to/maintain/tidb-in-kubernetes/tidb-node.md]:
```suggestion
- [`tkctl`](/tidb-in-kubernetes/tkctl.md)
```
[`tkctl`](/tidb-in-kubernetes/tkctl.md) is pending transaltion by @CaitinChen. This PR should be merged after this link works.

File: v3.0/how-to/maintain/tidb-in-kubernetes/tidb-node.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+# Maintain Kubernetes Nodes that Hold the TiDB Cluster
+
+TiDB is a highly available database that can run smoothly when some of the database nodes go offline. For this reason, you can safely shut down and maintain the Kubernetes nodes at the bottom layer without influencing TiDB's service. Specifically, you need to adopt various maintenance strategies when handling nodes that hold PD, TiKV, and TiDB instances because of their different features.
+
+This document introduces how to perform a temporary or long-term maintenance task for the Kubernetes nodes.
+
+## Prerequisites
+
+- [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [`tkctl`](/tidb-in-kubernetes/tkctl.md)
```
Comment (yikeke) [inline v3.0/how-to/maintain/tidb-in-kubernetes/tidb-node.md]: ditto

File: dev/how-to/maintain/tidb-in-kubernetes/tidb-node.md [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    watch pd-ctl -d store ${ID}
+    ```
+
+5. Unbind the TiKV instance from the local drive of the node:
+
+    Get the `PesistentVolumeClaim` used by the pod:
```
Comment (CaitinChen) [inline dev/how-to/maintain/tidb-in-kubernetes/tidb-node.md]:
```suggestion
    Get the `PesistentVolumeClaim` used by the Pod:
```

File: dev/how-to/maintain/tidb-in-kubernetes/tidb-node.md [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+
+6. Delete the TiKV instance:
+
+    {{< copyable "shell-regular" >}}
+    
+    ```shell
+    kubectl delete -n ${namespace} pod ${pod_name}
+    ```
+
+7. Check whether the TiKV instance is normally scheduled to other node:
```
Comment (CaitinChen) [inline dev/how-to/maintain/tidb-in-kubernetes/tidb-node.md]:
```suggestion
7. Check whether the TiKV instance is normally scheduled to another node:
```

File: dev/how-to/maintain/tidb-in-kubernetes/tidb-node.md [hunk 7]
```diff
@@ -0,0 +1,240 @@
...
+
+    {{< copyable "shell-regular" >}}
+    
+    ```shell
+    watch kubectl -n ${namespace} get pod -o wide
+    ```
+
+    If there are more TiKV instances on the maintenance node, you need to follow the above steps until all instances are migrated to other node.
```
Comment (CaitinChen) [inline dev/how-to/maintain/tidb-in-kubernetes/tidb-node.md]:
```suggestion
    If there are more TiKV instances on the maintenance node, you need to follow the above steps until all instances are migrated to other nodes.
```

File: dev/how-to/maintain/tidb-in-kubernetes/tidb-node.md [hunk 8]
```diff
@@ -0,0 +1,240 @@
...
+
+9. Confirm again that there is no more TiKV, TiDB and PD instances running on this node:
+
+    {{< copyable "shell-regular" >}}
+    
+    ```shell
+    kubectl get pod --all-namespaces | grep <node-name>
+    ```
+
+10. (optional) If this node is made offline for a long time, it is recommended to delete it from the Kubernetes cluster:
```
Comment (CaitinChen) [inline dev/how-to/maintain/tidb-in-kubernetes/tidb-node.md]:
```suggestion
10. (Optional) If this node is made offline for a long time, it is recommended to delete it from the Kubernetes cluster:
```

File: dev/how-to/maintain/tidb-in-kubernetes/tidb-node.md (hunk 3 above)
Comment (yikeke) [reply dev/how-to/maintain/tidb-in-kubernetes/tidb-node.md]: Got it.

**Comment by yikeke** [comment]
Comments addressed. @CaitinChen I will merge this PR after https://github.com/pingcap/docs/pull/1354 is merged.

**Pull request merged**